content/en/blog/_posts/2021-07-16-read-write-once-pod-access-mode-alpha.md

```diff
@@ -0,0 +1,267 @@
+---
+layout: blog
+title: "Introducing Single Pod Access Mode for PersistentVolumes"
+date: 2021-07-16
+slug: read-write-once-pod-access-mode-alpha
+---
+
+**Author:** Chris Henzie (Google)
+
+Kubernetes 1.22 introduces a new ReadWriteOncePod access mode for PersistentVolumes and PersistentVolumeClaims that allows users to restrict volume access to a single pod in the cluster.
+
+## What are access modes and why are they important?
+
+When using storage, there are different ways we can model how it is consumed.
+
+For example, a storage system like a network file share can have many users all reading and writing data simultaneously.
+In other cases maybe everyone is allowed to read data but not write it.
+For highly sensitive data, maybe only one user is allowed to read and write data but nobody else.
+
+In the world of Kubernetes, [access modes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes) are the way we model how storage is consumed.
+These access modes are a part of the spec for PersistentVolumes (PVs) and PersistentVolumeClaims (PVCs).
+
+```yaml
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: shared-cache
+spec:
+  accessModes:
+  - ReadWriteMany # Allow many pods to access shared-cache simultaneously.
+  resources:
+    requests:
+      storage: 1Gi
+  storageClassName: ""
+```
+
+In Kubernetes, we currently have three access modes for PVs and PVCs:
+
+- ReadWriteOnce -- the volume can be mounted as read-write by a single node
+- ReadOnlyMany -- the volume can be mounted read-only by many nodes
+- ReadWriteMany -- the volume can be mounted as read-write by many nodes
```

Comment:
nit: I would use an en dash here (`&ndash;`), rather than `--` (check the [preview](https://deploy-preview-29059--kubernetes-io-main-staging.netlify.app/blog/2021/07/16/read-write-once-pod-access-mode-alpha/) to see how these render)